scripts/r3/ingestion_under_fire.py

```
@@ -371,6 +650,317 @@ def _keys_for_scenario(candidate_sha: str, scenario: str, n: int) -> list[str]:
+
+    merged_status: dict[str, int] = {}
+    total_timeouts = 0
+    total_conn_errors = 0
+    for _ in range(len(configs)):
+        try:
+            r = result_queue.get(timeout=5)
+            for k, v in r["status_counts"].items():
+                merged_status[k] = merged_status.get(k, 0) + v
+            total_timeouts += r["timeouts"]
+            total_conn_errors += r["conn_errors"]
+        except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)